pages/index.tsx

```diff
@@ -104,6 +104,21 @@ const Home: NextPage = () => {
             </Box>
           </div>
         </SimpleGrid>
+        <Heading as="h2" mt="1rem" color="white.600">
+          {t('BADGE_STORE_HEADING')}{' '}
+        </Heading>
+        <NextLink href="/quests" locale={router.locale} passHref>
```

Comment:
It's hard to tell if it can be pressed or not when hovering, so it would be better to wrap it with a link component.
https://chakra-ui.com/docs/components/link
```
<NextLink>
  <Link>
    <image>
  <Link/>
<NextLink/>
```
```

Reply:
Yep! I will fix that once the badges page is pushed to main.

Reply:
I've merged the relevant PR!
https://github.com/henkaku-center/omise-interface/pull/102